arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -23,178 +23,227 @@
         ],
         "description": "Gets a failover group.",
         "operationId": "FailoverGroups_Get",
-        "x-ms-examples": {
-          "Get failover group": {
-            "$ref": "../examples/FailoverGroupGet.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
-          },
-          {
             "$ref": "#/parameters/SubscriptionIdParameter"
           },
           {
             "$ref": "#/parameters/ResourceGroupParameter"
           },
           {
-            "$ref": "#/parameters/ServerParameter"
+            "name": "serverName",
+            "in": "path",
+            "description": "The name of the server containing the failover group.",
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/FailoverGroupParameter"
+            "name": "failoverGroupName",
+            "in": "path",
+            "description": "The name of the failover group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
Consistent with newer specs we are introducing in the repo which we are making sure adhere to this convention. We're in the process of writing a validation rule to that effect. 
Simply put we want to ensure that the parameter section has path parameters first followed by body parameters and the client-level parameters (```api-version``` and ```subscriptionId```) in the beginning or at the end of the parameters section.